[2024-03-12T21:25:30.291-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_data.validate_train manual__2024-03-13T01:24:09.036247+00:00 [queued]>
[2024-03-12T21:25:30.315-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_data.validate_train manual__2024-03-13T01:24:09.036247+00:00 [queued]>
[2024-03-12T21:25:30.315-0400] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-03-12T21:25:30.349-0400] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): validate_train> on 2024-03-13 01:24:09.036247+00:00
[2024-03-12T21:25:30.363-0400] {standard_task_runner.py:60} INFO - Started process 407946 to run task
[2024-03-12T21:25:30.370-0400] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'train_data', 'validate_train', 'manual__2024-03-13T01:24:09.036247+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmpzh59epc9']
[2024-03-12T21:25:30.371-0400] {standard_task_runner.py:88} INFO - Job 93: Subtask validate_train
[2024-03-12T21:25:30.439-0400] {task_command.py:423} INFO - Running <TaskInstance: train_data.validate_train manual__2024-03-13T01:24:09.036247+00:00 [running]> on host DESKTOP-6U7QAQN.
[2024-03-12T21:25:30.571-0400] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='youremail@email.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_data' AIRFLOW_CTX_TASK_ID='validate_train' AIRFLOW_CTX_EXECUTION_DATE='2024-03-13T01:24:09.036247+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-13T01:24:09.036247+00:00'
[2024-03-12T21:25:31.214-0400] {logging_mixin.py:188} INFO - No anomalies detected in new data
[2024-03-12T21:25:31.216-0400] {python.py:202} INFO - Done. Returned value was: None
[2024-03-12T21:25:31.222-0400] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=train_data, task_id=validate_train, execution_date=20240313T012409, start_date=20240313T012530, end_date=20240313T012531
[2024-03-12T21:25:31.310-0400] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-12T21:25:31.343-0400] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
